mobile/library/java/org/chromium/net/impl/CronvoyBidirectionalStream.java

```diff
@@ -949,6 +949,11 @@ public void onSendWindowAvailable(EnvoyStreamIntel streamIntel) {
     }
   }
 
+  @Override
+  public void onDataAvailable(int bytes_available, EnvoyStreamIntel streamIntel) {
+    // TODO(abeyad) plumb through to YT
```

Comment:
Ideally we would want to update the builder API to be able to pass a custom logic for this, but since this may end up being an experimental code, it's probably okay to leave it as it is for now.

Reply:
I think we wanted to check in YT if YT thought there was no data available, but the ping indicated there was data available.

Reply:
right, and if YT thinks there was no data available but we get this callback, only way we'd know about it is if we create an error for it and stuff the info from bytes available and stream intel into the error detail message.  So we'd probably temporarily be creating more error entries than normal, which is fine.